src/grandorgue/model/GOSoundingPipe.cpp

```diff
@@ -486,17 +509,11 @@ void GOSoundingPipe::UpdateAudioGroup() {
 }
 
 void GOSoundingPipe::SetTemperament(const GOTemperament &temperament) {
+  m_IsTemperamentOriginalBased = temperament.GetIsTemperamentOriginalBased();
   if (!m_RetunePipe)
     m_TemperamentOffset = 0;
   else
-    m_TemperamentOffset = temperament.GetOffset(
-      m_PipeConfigNode.GetEffectiveIgnorePitch(),
-      m_MidiKeyNumber,
-      m_SoundProvider.GetMidiKeyNumber(),
-      m_SoundProvider.GetMidiPitchFract(),
-      m_HarmonicNumber,
-      m_PitchCorrection,
-      m_PipeConfigNode.GetDefaultTuning());
+    m_TemperamentOffset = temperament.GetOffset(m_MidiKeyNumber % 12);
```

Comment:
```suggestion
    m_TemperamentOffset = m_RetunePipe ? temperament.GetOffset(m_MidiKeyNumber % 12) : 0;
```

Then we can get rid of `if`.

Reply:
It's possible. But I won't do it. One thing is the pre-commit hook failing and the other is that ternary operator is inherently more unclear to read than the more verbose if ... else blocks.